wrapper/README.md

```diff
@@ -0,0 +1,67 @@
+# libuuu
+
```

Comment:
Can you add link in home ReadME file to like user jump to this file? 

Like py bind see ...

Reply:
Sure, will do.